[doc] Update kubernetes' FAQ (#10221)
apache/dolphinscheduler: docs/docs/en/guide/installation/kubernetes.md

> Because of the commercial license, we cannot directly use the driver of MySQL.
>
> If you want to use MySQL, you can build a new image based on the `apache/dolphinscheduler-<service>` image follow the following instructions:
> 

> > Since version 3.0.0, dolphinscheduler has been microserviced and the change of metadata storage requires replacing all services with MySQL driver.
> >
> > dolphinscheduler-tools dolphinscheduler-master dolphinscheduler-worker dolphinscheduler-api dolphinscheduler-alert-server

Reviewer: ```suggestion
> Since version 3.0.0, dolphinscheduler has been microserviced and the change of metadata storage requires replacing all services with MySQL driver, which including dolphinscheduler-tools, dolphinscheduler-master, dolphinscheduler-worker, dolphinscheduler-api, dolphinscheduler-alert-server
```

> Because of the commercial license, we cannot directly use the driver of MySQL.
>
> If you want to add MySQL datasource, you can build a new image based on the `apache/dolphinscheduler-<service>` image follow the following instructions:
> 

> > You needs to change the two images of dolphinscheduler-worker dolphinscheduler-api service.

Reviewer: ```suggestion
> You need to change the two service images including dolphinscheduler-worker, dolphinscheduler-api.
```

1. Download the MySQL driver [mysql-connector-java-8.0.16.jar](https://repo1.maven.org/maven2/mysql/mysql-connector-java/8.0.16/mysql-connector-java-8.0.16.jar).

2. Create a new `Dockerfile` to add MySQL driver:

```
FROM dolphinscheduler.docker.scarf.sh/apache/dolphinscheduler-<service>:<version>
# For example

> # FROM dolphinscheduler.docker.scarf.sh/apache/dolphinscheduler-tools:<version>

Reviewer: We should use two of the docker image we metion above
```suggestion
# FROM dolphinscheduler.docker.scarf.sh/apache/dolphinscheduler-worker:<version>
```

> Because of the commercial license, we cannot directly use the driver of Oracle.
>
> If you want to add Oracle datasource, you can build a new image based on the `apache/dolphinscheduler-<service>` image follow the following instructions:
>

> > You needs to change the two images of dolphinscheduler-worker dolphinscheduler-api service.

Reviewer: could we combine MySQL and Oracle into the same section? I find out the only different is jar name, WDYT?

Author: Sure
